### DISCUSSION TITLE
awscloudwatchmetrics receiver

### DISCUSSION THREAD

**Discussion by jaslkaur29** [open-telemetry/opentelemetry-collector-contrib#31391] [Q&A]
Hi, I wanted to know when will the awscloudwatchmetrics be added to the contrib distribution? I tried to use this receiver and got an error decoding 'receivers': unknown type: "awscloudwatchmetrics" for id: "awscloudwatchmetrics"

**Comment by jaslkaur29** [comment]
@jpkrohling Will you be able to help with this query? Thank you

**Comment by jpkrohling** [comment]
It looks like it wasn't fully implemented yet.
